src/main/java/domain/AttendTime.java

```diff
@@ -1,23 +1,52 @@
 package domain;
 
-import java.time.LocalDateTime;
-import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+import java.time.LocalTime;
 
 public class AttendTime {
 
-    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+    private final LocalDate localDate;
+    private final LocalTime localTime;
+    public AttendTime(LocalDate localDate,LocalTime localTime) {
+        validate(localDate);
+      this.localDate=localDate;
+      this.localTime=localTime;
+    }
+
+    public void validate(LocalDate localDate) {
+        validateWeekend(localDate);
+        validateHoliday(localDate);
+    }
+
+    private void validateWeekend(LocalDate localDate) {
+        if (localDate.getDayOfWeek().getValue() > 5)
+            throw new IllegalArgumentException("주말은 출석 할 수 없습니다.");
+    }
 
-    private final LocalDateTime attendTime;
+    private void validateHoliday(LocalDate localDate) {
+        if (December.checkHoliday(localDate.getDayOfMonth())) {
+            throw new IllegalArgumentException("휴일은 출석 할 수 없습니다.");
+        }
+    }
+
+    public AttendanceStatus checkAttendanceStatus() {
+        int startingHour = AttendTimeOfWeekDay.getTimeByDayOfWeekDay(localDate.getDayOfWeek().getValue());
+        return AttendanceStatus.calculateStatus(LocalTime.of(startingHour, 5), LocalTime.of(startingHour, 30), localTime);
+    }
+
+    public boolean checkSameDate(int date) {
+        return localDate.getDayOfMonth()==date;
```

Comment:
```suggestion
        return localDate.getDayOfMonth() == date;
```
코드 전체적으로 [Java Google Style Guide](https://github.com/woowacourse/woowacourse-docs/tree/main/styleguide/java)를 잘 적용했는지 확인해볼까요?
띄워쓰기 같은 경우는 IntelliJ의 도움을 받으면 보다 손쉽게 적용할 수 있습니다. [참고](https://eroul-ri.tistory.com/26)

Reply:
Java google style guide를 적용하고 있었는데 놓친 부분도 있었군요.

저렇게 '=='을 코드에 이어 붙히면 정렬 키를 눌러도 정렬되지 않는 것 같습니다. 
getDayOfMonth() == 앞에 한칸이라도 떨어져 있다면 뒷 부분도 자동 정렬해주지만,
앞부분이 붙어있다면 체크하지 못하는 것 같습니다.

이 부분도 새롭게 알게 되었습니다!

꼼꼼하게 체크해주셔서 감사합니다!